src/main/java/com/example/training/controller/AmazonController.java

```diff
@@ -0,0 +1,42 @@
+package com.example.training.controller;
+
+import com.example.training.service.AmazonS3Service;
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+
+@RestController
+@RequestMapping("amazon")
+@RequiredArgsConstructor
+public class AmazonController {
+    private final AmazonS3Service amazonS3Service;
+
+    @GetMapping("download/{fileName}")
```

Comment:
have a look at best practices with naming REST endpoints. should be /files/{fileName}